packages/truffle-codec-utils/src/abi.ts

```diff
@@ -53,6 +53,7 @@ export namespace AbiUtils {
     type: string;
     indexed?: boolean; //only present for inputs
     components?: AbiParameter[]; //only preset for tuples (structs)
+    internalType?: string;
```

Comment:
What is "internal type"?

Reply:
It's the new field added in 0.5.11 which I earlier went and added to `truffle-contract-schema`.  It's basically a type hint for parameters which have types which lose a bit when converted to ABI.  E.g., for a struct or an enum, it'll tell you the name of the struct or the enum.  My equivalent for it in the decoder output format is called `typeHint`.

Reply:
Ah right, I remember that now. Thanks!